lib/utils/string.ts

```diff
@@ -29,21 +29,21 @@ export const toCamelCase = (string: string): string =>
   string.toLowerCase().replace(/((?:-|_|\s)\w)/g, (k: string) => k[1].toUpperCase());
 
 /**
- * Convert the string to PascalCase
+ * Converts the first character in the string to an uppercase equivalent.
  * @param string
  */
-export const toPascalCase = (string: string): string =>
-  capitalize(string.toLowerCase().replace(/((?:-|_|\s)\w)/g, (k: string) => k[1].toUpperCase()));
+export const capitalize = (string: string): string => string.charAt(0).toUpperCase() + string.slice(1);
 
 /**
- * Converts the first character in the string to an uppercase equivalent.
+ * Convert the string to PascalCase
  * @param string
  */
-export const capitalize = (string: string): string => string.charAt(0).toUpperCase() + string.slice(1);
+export const toPascalCase = (string: string): string =>
+  capitalize(string.toLowerCase().replace(/((?:-|_|\s)\w)/g, (k: string) => k[1].toUpperCase()));
 
 /**
  * Converts the first character in the string, and the first character after un _, to a lowercase equivalent.
  * @param string
  */
-export const uncapitalize = (string: string): string =>
+export const unCapitalize = (string: string): string =>
```

Comment:
()The _uncapitalise_ word does not sound good. Could we rename the function to something like `firstLetterToLowerCase()` or `firstToLowerCase()`?